hips/hip-9999.md

```diff
@@ -0,0 +1,240 @@
+---
+hip: 9999
+title: "New annotations for displaying job output"
+authors: [ "Ian Zink <ian@replicated.com>" ]
+created: "2023-01-26"
+type: "feature"
+status: "draft"
+---
+
+## Abstract
+
+This proposes a new annotation to indicate that the output from the job should be displayed to the user.
```

Comment:
In this case, I think you mean the output of a hook not a job. A hook may be implemented with a job but it's the hook.

For example, a job that's not a hook would not be linked to this annotation. I think the title needs to reflect any change here.